apps/io-wallet-user-func/src/app/config.ts

```diff
@@ -55,10 +55,16 @@ export const MailConfig = t.type({
   mailSender: NonEmptyString,
   mailupSecret: NonEmptyString,
   mailupUsername: NonEmptyString,
+  walletInstanceRevocationEmailFeatureFlag: t.boolean,
 });
 
 export type MailConfig = t.TypeOf<typeof MailConfig>;
 
+export const WALLET_REVOCATION_EMAIL_TITLE =
+  "Messaggi da IO: IT Wallet disattivato";
```

Comment:
```suggestion
  "Messaggio da IO: Documenti su IO disattivato";
```
and you should move it in the handler